pkg/client/userd/trafficmgr/podaccess.go

```diff
@@ -0,0 +1,272 @@
+package trafficmgr
+
+import (
+	"context"
+	"fmt"
+	"net/netip"
+	"sync"
+	"time"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/durationpb"
+
+	"github.com/datawire/dlib/dgroup"
+	"github.com/datawire/dlib/dlog"
+	"github.com/telepresenceio/telepresence/rpc/v2/daemon"
+	"github.com/telepresenceio/telepresence/v2/pkg/agentconfig"
+	"github.com/telepresenceio/telepresence/v2/pkg/client"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/remotefs"
+	"github.com/telepresenceio/telepresence/v2/pkg/forwarder"
+	"github.com/telepresenceio/telepresence/v2/pkg/iputil"
+)
+
+type podAccess struct {
+	// ctx is a context canceled by the cancel attribute. It must be used by
+	// services that should be canceled when the intercept ends
```

Comment:
a nitpick, but some comments in this file weren't updated when after the move
```suggestion
	// services that should be canceled when the intercept/ingest ends
```